spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/pubsub/it/PubSubTemplateIntegrationTests.java

```diff
@@ -106,35 +117,53 @@ public void testPullAndAck() {
 			String topicName = "peel-the-paint" + UUID.randomUUID();
 			String subscriptionName = "i-lost-my-head" + UUID.randomUUID();
 			pubSubAdmin.createTopic(topicName);
-			pubSubAdmin.createSubscription(subscriptionName, topicName);
+			pubSubAdmin.createSubscription(subscriptionName, topicName, 10);
```

Comment:
Can we lower it to like 3 seconds, so that the test runs faster?

Reply:
no, 10 is the minimum value